test/Graphics/Drawable.test.cpp

```diff
@@ -23,14 +23,14 @@ class DrawableTest : public sf::Drawable
     mutable int m_callCount{};
 };
 
-TEST_CASE("[Graphics] sf::Drawable", runDisplayTests())
+TEST_CASE("[Graphics] sf::Drawable")
```

Comment:
You need to keep the `runDisplayTests()` as not all CI builds support tests that require a display